2-api/key-concepts.md

```diff
@@ -2,71 +2,84 @@
 
 # Key concepts
 
-To best understand the reference documentation, you must first know these key concepts.
+To best understand the reference documentation, you must first understand these
+key concepts.
 
 ## Attribute
 
-A comment label used for review. The most popular attributes are the toxicity and severe toxicity models.
+An **attribute** is a comment label used for review. The most popular attribute
+is "toxicity".
+
+This was previously known also as "model".
 
 ## Comment
 
 A **comment** is the text to be scored. Each API request contains a single
 comment. A comment could be a single post to a web page's comments section,
 a forum post, a message to a mailing list, a chat message, etc.
 
-## Model
+## Context (coming soon)
+
+_(Coming soon)_ **Context** is a representation of the conversation context
+for the comment, such as an article that is being commented on, or another
+comment that is being replied to. It is used in the analysis of the
+comment text, such as when determining the "off-topic" model score.
 
- A **model** is a dimension that the comment is scored on; for example,
- "toxic," "obscene," "thoughtful," "off-topic," etc. This is also sometimes referred to
- as an **attribute** or **model attribute**.
+**Note**: you can send context with your requests, however our current models
+are not making use of it. At this time, sending context won't change a model's
+scores.
 
-### Model attribute scores
+## Method
 
-The API can return **model attribute scores** in different formats, known as
-**score types**. Currently, the only score type supported is a probability score
-between 0 and 1, with higher values indicating greater likelihood of the attribute
-label.
+A **method** is a procedure associating a message (or request) with an
+object (which consists of data and behavior). A method request will return
+a new object.
 
-### Summary Score
+For example, the [`AnalyzeComment` method](methods.md#scoring-comments-analyzecomment)
+sends a request to score the comment object. The comment object includes
+comment text, context, and more. The method then return the attribute span
+scores.
 
-A model's **summary score** provides an overall score for the entire
-comment. While the API may return multiple span scores for a given input, it
-will always return exactly one summary score.
+
+### Score types
+
+**Score types** are different formats for the API to return model attribute
+scores. Currently, the only score type supported is the probability score
+type with a value between 0 and 1. Higher values indicating greater likelihood
```

Comment:
I don't think this is technically true. Users can request the RAW score, though the results of it aren't very meaningful since they aren't calibrated.

Reply:
Should I just remove the last sentence?